packages/2d/src/lib/components/Media.ts

```diff
@@ -50,6 +48,7 @@ export abstract class Media extends Asset {
     if (props.play) {
       this.play();
     }
+    this.volume = props.volume || 1;
```

Comment:
I think this breaks setting volume to 0, should probably instead be
```suggestion
    this.volume = props.volume ?? 1;
```
in order to not overwrite volume 0

Reply:
you're right, thank you!